app.js

```diff
@@ -108,17 +109,30 @@ function countClicks(event) {
   for(var i = 0; i < pictureArray.length; i++){
     if(event.target.name === pictureArray[i].name)
       pictureArray[i].numberOfTimesClicked++;
+    // picturesStorageData();
       // console.log(pictureArray[i].numberOfTimesClicked);
   }
   if (totalNumberOfClicks < 25){
     generateNewImages();
+    picturesStorageData();
```

Comment:
You might want to save the click data for the final click a user makes as well.  Currently the data is only stored 0-24 clicks but not the 25th.